changedetectionio/update_worker.py

```diff
@@ -78,6 +78,8 @@ def send_content_changed_notification(self, t, watch_uuid):
                 'screenshot': watch.get_screenshot_as_jpeg() if watch.get('notification_screenshot') else None,
                 'current_snapshot': snapshot_contents.decode('utf-8'),
                 'diff': diff.render_diff(watch_history[dates[-2]], watch_history[dates[-1]], line_feed_sep=line_feed_sep),
+                'diff_added': diff.render_diff(watch_history[dates[-2]], watch_history[dates[-1]], False, False, True, line_feed_sep=line_feed_sep),
```

Comment:
one thing on these few lines, could you call it with the argument name
`render_add=True` etc

Reply:
done